cmd/juju/cloud/remove.go

```diff
@@ -117,12 +117,35 @@ func (c *removeCloudCommand) Run(ctxt *cmd.Context) error {
 }
 
 func (c *removeCloudCommand) removeLocalCloud(ctxt *cmd.Context) error {
+	cloudDetails, err := listLocalCloudDetails(c.Store)
+	if err != nil {
+		return err
+	}
+	if _, ok := cloudDetails.personal[c.Cloud]; ok {
+		return c.removeLocalPersonalCloud(ctxt)
+	}
+	if _, ok := cloudDetails.builtin[c.Cloud]; ok {
+		ctxt.Infof("Cannot remove built-in cloud %q", c.Cloud)
```

Comment:
can we add "from client" to this? it's a bit confusing when using remove-cloud interactively with choice 3

```
juju remove-cloud oracle
This operation can be applied to both a copy on this client and to the one on a controller.
Do you want to remove cloud oracle from:
    1. client only (--client)
    2. controller "tuesday" only (--controller tuesday)
    3. both (--client --controller tuesday)
Enter your choice, or type Q|q to quit: 3
Cannot remove public cloud "oracle"
To hide this cloud, remove it's credentials with `juju remove-credential`
No cloud called "oracle" exists on controller "tuesday"
```